tests/imposter/private-api-config.yaml

```diff
@@ -12,73 +12,112 @@ system:
       preloadFile: data/questions.json
 
 resources:
+  ### "single-question" journey ###
   - method: GET
     path: /question
     requestHeaders:
-      session-id: "success"
+      session-id: "single-question"
```

Comment:
Wondering can we wildcard this input somehow, rather than having to define these endpoints for each of the 'journeys'?

Reply:
I did spend most of yesterday trying to get that to work, but the framework only allows us to wildcare/optional match fields in the request body, not requestHeaders 😢

Reply:
Might get a little unwieldy but should be manageable